package.yaml

```diff
@@ -0,0 +1,98 @@
+name:       apk-build
+version:    0.0.0.1
+synopsis:   Utilities to build APK packages
+homepage:   https://github.com/mbj/apk-build
+author:     Markus Schirp
+maintainer: mbj@schirp-dso.com
+copyright:  2019 Markus Schirp
+github:     mbj/apk-build
+
+dependencies:
+- amazonka                == 1.6.1
```

Comment:
You don't want to use the `stack-dependencies.txt` scheme with these?

Reply:
Only for the ones I can see being low maintenance / floating. Amazonka definitively does not fall into that class.